front/app/components/SeatInfo/BillingInfo.tsx

```diff
@@ -0,0 +1,174 @@
+import useSeats from 'api/seats/useSeats';
+
+// types
+import { SeatInfoProps, SeatNumbersType, SeatTypeMessageDescriptor } from '.';
+
+// Intl
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// Utils
+import { isNil } from 'utils/helperUtils';
+import Link from 'utils/cl-router/Link';
+
+const StyledLink = styled(Link)`
+  text-decoration: underline;
+
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const BillingInfo = ({ seatType }: SeatInfoProps) => {
+  const { formatMessage } = useIntl();
+  const { pathname } = useLocation();
+  const { data: appConfiguration } = useAppConfiguration();
+  const { data: seats } = useSeats();
+  const adminsAndMangersLink = '/admin/users/admins-managers';
+  const isOnAdminsAndMangersPage = pathname.includes(adminsAndMangersLink);
+
+  const maximumSeatNumbers: SeatNumbersType = {
+    admin:
+      appConfiguration?.data.attributes.settings.core.maximum_admins_number,
+    collaborator:
+      appConfiguration?.data.attributes.settings.core.maximum_moderators_number,
+  };
+  const maximumSeatNumber = maximumSeatNumbers[seatType];
+
+  const additionalSeatNumbers: SeatNumbersType = {
+    admin:
+      appConfiguration?.data.attributes.settings.core.additional_admins_number,
+    collaborator:
+      appConfiguration?.data.attributes.settings.core
+        .additional_moderators_number,
+  };
+  const maximumAdditionalSeats = additionalSeatNumbers[seatType];
+
+  // Maximum seat number being null means that there are unlimited seats so we don't show the seat info
+  if (isNil(maximumSeatNumber) || isNil(maximumAdditionalSeats) || !seats) {
+    return null;
+  }
+
+  const usedSeats = {
+    admin: seats.data.attributes.admins_number,
+    collaborator: seats.data.attributes.project_moderators_number,
+  }[seatType];
+  const seatTypeMessage: SeatTypeMessageDescriptor = {
+    admin: messages.adminSeats,
+    collaborator: messages.collaboratorSeats,
+  };
+  const seatTypeTitle = seatTypeMessage[seatType];
+  const seatTypeTooltipMessages: SeatTypeMessageDescriptor = {
+    admin: messages.adminSeatsTooltip,
+    collaborator: messages.collaboratorSeatsTooltip,
+  };
+  const seatTypeTooltipMessage = seatTypeTooltipMessages[seatType];
+  const totalSeats = maximumAdditionalSeats + maximumSeatNumber;
+  const remainingSeats = totalSeats - usedSeats;
+
+  let totalSeatsBreakdownMessage = formatMessage(messages.seatsWithinPlanText);
+  if (usedSeats > maximumSeatNumber) {
+    // Any used seats exceeding the maximum set seat number are additional seats
+    const noOfAdditionalSeats = usedSeats - maximumSeatNumber;
+    totalSeatsBreakdownMessage = formatMessage(messages.seatsExceededPlanText, {
+      noOfSeatsInPlan: maximumSeatNumber,
+      noOfAdditionalSeats,
+    });
+  }
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      padding="20px"
+      bgColor={rgba(colors.teal400, 0.07)}
```

Comment:
Still maintaining this `rgba` usage from the original since we can't use a direct one from storybook. Also keeping the changes minimal from already existing work